credsweeper/credentials/line_data.py

```diff
@@ -112,7 +112,9 @@ def clean_bash_parameters(self) -> None:
 
     def sanitize_variable(self) -> None:
         """Remove trailing spaces, dashes and quotations around the variable."""
-        if self.variable:
+        mood_var_len = 0
```

Comment:
May I ask why you named variable name like `mood_var_len`?

Reply:
My fail in grammatik :)